### PR TITLE
Drop support for PHP7.4, Symfony version upgraded to 5.4

### PR DESCRIPTION
**Pull request by Rafikooo**

### CODE REVIEW

**Comment by Roshyo** [comment]
Why dropping support for 7.4 ? It is still supported by Sylius

**Comment by lchrusciel** [comment]
Hey Stephane, you are correct. Yet, it is supported only by previous releases, and we want to move forward with the whole community. Therefore, all future releases will have PHP^8.0 and SF ^4.4 and ^5.4 support only

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -17,19 +17,13 @@ jobs:
...
-                    -
-                        php: 7.4
-                        symfony: 4.4.*
-                    -
-                        php: 7.4
-                        symfony: 5.1.*
                     -
                         php: 8.0
                         symfony: 4.4.*
                         composer-flags: --ignore-platform-req=php
```
Comment (lchrusciel) [inline .github/workflows/build.yml:23]: Can we drop these flags?

Comment (Rafikooo) [reply .github/workflows/build.yml]: Both flags dropped

**Review by Zales0123 — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Rafał! :1st_place_medal:

**Pull request merged**